### PR TITLE
Make player tumbling configurable

### PR DESCRIPTION
**Pull request by mwolting**
This disables tumbling of the player by default, and makes it configurable through the game properties.

<feature

### CODE REVIEW

**Comment by NilsHullegien** [comment]
This will improve the playability of the game, but still keep the extreme death screen. 
Also a good idea to set it up as a config!
Code looks good.

<review

File: core/src/main/java/com/group5/core/physics/PlayerPhysicsStrategy.java [hunk 1]
```diff
@@ -84,7 +86,8 @@ public void jump(final float intensity) {
      */
     public void kill() {
         Body b = getBody();
-        World.setVelocityThreshold(100000000);
+        b.setFixedRotation(false);
+        World.setVelocityThreshold(Integer.MAX_VALUE);
```
Comment (levilime) [inline core/src/main/java/com/group5/core/physics/PlayerPhysicsStrategy.java]: Nice that you replaced the magic number with this, more descriptive of it's use this way.

File: desktop/src/main/java/com/group5/java/EndlessRunnerDesktop.java [hunk 2]
```diff
@@ -14,6 +14,7 @@
      */
     public static void main(final String[] args) {
         LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
+        config.title = "\uD83D\uDC14";
```
Comment (levilime) [inline desktop/src/main/java/com/group5/java/EndlessRunnerDesktop.java]: This emoticon does not work on my Windows 7 pc.

Comment (levilime) [inline desktop/src/main/java/com/group5/java/EndlessRunnerDesktop.java]: This is a chicken emoticon, please add as a named variable or put comment to make this clear to possible code reader.

**Comment by levilime** [comment]
I agree with this merge functionally, could you please clarify the chicken emoticon for the reader <review

**Pull request merged**